fms/utils/generation.py

```diff
@@ -126,6 +135,7 @@ def generate(
     num_beams: int = 1,
     use_cache: bool = False,
     contiguous_cache: bool = False,
+    dynamic_cache: bool = False,
```

Comment:
Can this be found from the `torch._dynamo.config.dynamic_shapes`. This way a user does not need to pass this

Reply:
I was going to do this at the script level, to leave some control to the user. Someone might want to keep the default torch.compile() behavior to measure compilation times.